context/app/static/js/shared-styles/dropdowns/DropdownListboxOption/style.js

```diff
@@ -0,0 +1,21 @@
+import styled from 'styled-components';
+import MenuItem from '@material-ui/core/MenuItem';
+import CheckRoundedIcon from '@material-ui/icons/CheckRounded';
+
+const iconSize = '1.25rem';
+
+const FlexMenuItem = styled(MenuItem)`
+  display: flex;
+`;
+
+const CheckIcon = styled(CheckRoundedIcon)`
+  font-size: ${iconSize};
+  margin-right: ${(props) => props.theme.spacing(1)}px;
+`;
+
+const StyledSpan = styled.span`
+  ${(props) => !props.isSelected && `padding-left: calc(${iconSize} + ${props.theme.spacing(1)}px)`};
```

Comment:
Sorry, I feel like I commented on something like this before: Do you think it would be possible to pull this out, perhaps add a class so more of the logic can be handled by CSS? It's just a pretty complicated line. No worries if nothing better strikes you.

Reply:
Using a class does nothing but change the selector, `padding-left: calc(${iconSize} + ${props.theme.spacing(1)}px)` won't change, so does using `.selected-class { ... }` instead of `${(props) => !props.isSelected &&` make it much more clear?

Reply:
A little more clear? I guess I'm just worried by they wrapped contexts here: The "padding-left" inside a boolean short circuit inside a function inside an interpolated string... but I might be coming with this with a set of experiences where more logic was put into the CSS, because we weren't building it in the code.

If you think this is readable for a contemporary audience, I won't argue it further.

Reply:
Props are primarily used for styled components, so i'd prefer to keep it this way. I do think it's a little cluttered, but I think this is the CSS in JS approach.